lnrpc/routerrpc/router.proto

```diff
@@ -343,14 +343,26 @@ message TrackPaymentsRequest {
 
 message RouteFeeRequest {
     /*
-    The destination once wishes to obtain a routing fee quote to.
+    The destination one wishes to obtain a routing fee quote to.
     */
     bytes dest = 1;
 
     /*
     The amount one wishes to send to the target destination.
     */
     int64 amt_sat = 2;
+
+    /*
+    A payment request of the target node that the route fee request is intended
+    for. Its parameters are input to probe payments that estimate routing fees.
+     */
+    string payment_request = 3;
+
+    /*
+    A user preference of how long a probe payment should maximally be allowed to
+    take, denoted in seconds.
+     */
+    int32 timeout = 4;
```

Comment:
The `SendPaymentRequest` tracks the timeout in seconds, hence I chose the same unit here.

Reply:
Yeah and I also doubt we'll ever want a timeout that is shorter than a second considering global E2E round trip single TCP communication is 150ms. This is with no packet loss or without considering situations where we need to have multiple round trips where we try multiple routes. And since this latency is primarily bound by the speed of light, I don't anticipate it changing 😁

Reply:
+1 for uint32